### PR TITLE
IPythonInterpreter delete temp file and close stream

### PR DESCRIPTION
**Pull request by yx91490**
### What is this PR for?
delete temporary file  and  close inputstream in IPythonInterpreter.checkIPythonPrerequisite()

### What type of PR is it?
[Bug Fix]

### Todos
*no

### What is the Jira issue?
* https://jira.apache.org/jira/browse/ZEPPELIN-3910, [ZEPPELIN-3910]

### How should this be tested?
* manual.

### Questions:
* Does the licenses files need update?
no
* Is there breaking changes for older versions?
no
* Does this needs documentation?
no

### CODE REVIEW

**Comment by zjffdu** [comment]
+1

File: python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java [hunk 1]
```diff
@@ -161,38 +161,46 @@ public void open() throws InterpreterException {
...
-        return "ipykernel is not installed";
-      }
-      if (!freezeOutput.contains("ipython=")) {
-        return "ipython is not installed";
-      }
-      if (!freezeOutput.contains("grpcio=")) {
-        return "grpcio is not installed";
+        try (FileInputStream in = new FileInputStream(stderrFile)) {
+          return "Fail to run pip freeze.\n" + IOUtils.toString(in);
+        }
```
Comment (felixcheung) [inline python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java]: is it going to close FileInputStream when it is out of scope here?

Comment (yx91490) [reply python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java]: yes

**Comment by Leemoonsoo** [comment]
LGTM

**Pull request closed without merging**